keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -463,6 +473,7 @@ high level (needs more precise definitions) those may be things like:
     job creation time) for cron job <= 10%
   - 99,9% of /health requests per day finish with 200 code
 -->
+All the `Available` pods created should be more than the time specified in `.spec.minReadySeconds` 99.99% of the time.
```

Comment:
how can a cluster-admin know this is happening?  Or more generally, if the statefulset controller starts failing, is there a metric that tracks errors from the statefulset controller that could be alerted upon if it starts failing very often?

Reply:
Based on the above metric.

Reply:
It's not clear how to track this from the description of the above metric. Please be more specific.

Furthermore, I don't think this SLO is appropriate. There's nothing in the KEP that suggests this is engineered for (or that we can accurately measure) 99.99% correctness.

I think it's reasonable to think about expected latencies and correctness here, but 99.99% is a very stringent target and this SLO as written doesn't seem universal.

You may want to take a look at the updated question template: https://github.com/kubernetes/enhancements/blame/master/keps/NNNN-kep-template/README.md#L512-L513

Reply:
I changed the SLO to 99% instead of 4 9's. I also added a sentence in the previous question on how the metric can be used to determine correctness of the functionality.